src/components/toolbar/ToolBar.vue

```diff
@@ -22,35 +27,43 @@
           >
             <font-awesome-icon :icon="panelsCompressed ? expandIcon : compressIcon" />
           </b-button>
-        </div>
-        <div class="btn-group btn-group-sm" role="group" aria-label="Publish controls">
+        </div> -->
+
+        <div
+          class="d-flex align-items-center btn-group btn-group-sm"
+          role="group"
+          aria-label="Publish controls"
+          data-cy="publish-control"
```

Comment:
Should `data-cy` be used instead of `data-test` for e2e testing? Or does `data-cy` have another use?

Reply:
Both properties are used for testing purposes